app/src/main/java/fr/free/nrw/commons/Utils.java

```diff
@@ -117,10 +117,10 @@ public static String fixExtension(String title, String extension) {
     public static void rateApp(Context context) {
         final String appPackageName = BuildConfig.class.getPackage().getName();
```

Comment:
oh, this should definitely be changed, it is a bad example that led you astray. `context.getPackageName()`